packages/sdk/src/utils/chain.ts

```diff
@@ -0,0 +1,101 @@
+import { BigNumber, ethers } from 'ethers'
+import { EthereumProvider } from 'hardhat/types'
+
+import { Chain, L1Chain } from '../interfaces'
+
+export const OptimismKovan = {
+  chainId: Chain.OPTIMISM_KOVAN,
+  chainName: 'OptimismKovan',
+  isTestChain: true,
+  isLocalChain: false,
+  multicallAddress: '0xE71bf4622578c7d1526A88CD3060f03030E99a04',
+  getExplorerAddressLink: (address: string) =>
+    `https://kovan-optimistic.etherscan.io/address/${address}`,
+  getExplorerTransactionLink: (transactionHash: string) =>
+    `https://kovan-optimistic.etherscan.io/tx/${transactionHash}`,
+} as const
+
+export const Optimism = {
+  chainId: Chain.OPTIMISM,
+  chainName: 'Optimism',
+  isTestChain: false,
+  isLocalChain: false,
+  multicallAddress: '0x35A6Cdb2C9AD4a45112df4a04147EB07dFA01aB7',
+  getExplorerAddressLink: (address: string) =>
+    `https://optimistic.etherscan.io/address/${address}`,
+  getExplorerTransactionLink: (transactionHash: string) =>
+    `https://optimistic.etherscan.io/tx/${transactionHash}`,
+} as const
+
+export const L1ToL2Mapping = {
+  [Chain.MAINNET]: Chain.OPTIMISM,
+  [Chain.KOVAN]: Chain.OPTIMISM_KOVAN,
+} as const
+
+export const L2ToL1Mapping = {
+  [Chain.OPTIMISM]: Chain.MAINNET,
+  [Chain.OPTIMISM_KOVAN]: Chain.KOVAN,
+} as const
+
+export const addOptimismNetworkToMetamask = async <
```

Comment:
Is this specific to MM? If not, maybe "addOptimismNetworkToWallet"

Reply:
To the best of my knowledge this only works on metamask.  I see here they are trying to make it a standard (it's status is stagnant) but I don't know if anybody else has picked it up.  https://eips.ethereum.org/EIPS/eip-3085